pkg/controller/oam/applicationconfiguration/apply.go

```diff
@@ -134,25 +141,9 @@ func findDereferencedScopes(statusScopes []v1alpha2.WorkloadScope, scopes []unst
 	return toBeDeferenced
 }
 
-func (a *workloads) applyScope(ctx context.Context, wl Workload, s unstructured.Unstructured) error {
-	workloadRef := runtimev1alpha1.TypedReference{
-		APIVersion: wl.Workload.GetAPIVersion(),
-		Kind:       wl.Workload.GetKind(),
-		Name:       wl.Workload.GetName(),
-	}
-
-	// Get Scope again to make sure we work with the most up-to date object.
-	scopeObject := unstructured.Unstructured{}
```

Comment:
I looked back in the code and realized this is not needed anymore. Thanks for removing it.